projects/map/src/lib/map-service/map.service.ts

```diff
@@ -158,6 +158,17 @@ export class MapService {
     return this.map.getProjection$().pipe(map(p => p.getCode()));
   }
 
+  /** Gets the UOM of the map as defined in the projection.
+   * Will return 'm' in case the map's projection has not defined UOM (such as EPSG:28992).
```

Comment:
well... either OL does not parse the proj string entirely or something else is going on because the definition we use does have a `units` dimension:
`EPSG:28992[+proj=sterea +lat_0=52.15616055555555 +lon_0=5.38763888888889 +k=0.9999079 +x_0=155000 +y_0=463000 +ellps=bessel +towgs84=565.237,50.0087,465.658,-0.406857,0.350733,-1.87035,4.0812 +units=m +no_defs]`